be/src/vec/sink/vtablet_sink.cpp

```diff
@@ -852,6 +852,10 @@ void VNodeChannel::cancel(const std::string& cancel_msg) {
     // But do we need brpc::StartCancel(call_id)?
     _cancel_with_msg(cancel_msg);
 
+    if (_stub == nullptr) {
```

Comment:
_stub is created in VNodeChannel::init, if _stub is null, it will set _is_closed to true.
We should not reach here normally.

If we do not find the root case, please add some comments and logs here.

Reply:
This was initially fixed in #30915 and later changed in #33006.
There is no more additional information in the latest Jira CIR-8301.

Reply:
I agree with @cambyzju. We should not reach here.

Reply:
`cancel()` could be called when `_inited = false`, see #34897 (fixed in that PR)